packages/ui-react/src/containers/AccountModal/forms/PersonalDetails.tsx

```diff
@@ -142,6 +147,7 @@ const PersonalDetails = () => {
   const { setValue, handleSubmit, handleChange, values, errors, validationSchemaError, submitting } = useForm<PersonalDetailsFormData>({
     initialValues,
     onSubmit: PersonalDetailSubmitHandler,
+    isReadyForInit: !isLoading,
```

Comment:
Maybe the form init should just happen when the whole data is available? Somewhere in the child component which is rendered after everything is loaded.

Reply:
Good idea.